packages/vdom-extension/src/index.tsx

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import { Kernel, Session } from '@jupyterlab/services';
```

Comment:
I was about to suggest you explicitly import these from services to at least make this dependency clear (as opposed to just rendermime-interfaces), then I saw you already did it!